katalog/metrics-server/README.md

```diff
@@ -8,13 +8,13 @@ on each node.
 
 ## Requirements
 
-- Kubernetes >= `1.10.0`
+- Kubernetes >= `1.0.10`
```

Comment:
Is it a typo?

Reply:
Yes, it is. My mind read Kustomize instead of Kubernetes. 😅

Reply:
Fixed. I adjusted the recommended version according to https://github.com/kubernetes-sigs/metrics-server/blob/master/README.md.